cs/configuring.texy

```diff
@@ -345,3 +344,17 @@ Například takto může vypadat definice služby pojmenované `database`, což
 services:
 	database: PDO('mysql:host=127.0.0.1;dbname=test', root, password)
 \--
+
+
+Rozšíření
+=========
+
+Pomocí rozšíření můžeme rozdělit naši aplikaci do malých nezavislých častí s vlastní konfigurací. Nette obsahuje řadu předdefinovaných rozšíření, o kterých se dočtete více v kapitole [DI: Základní rozšíření pro kontejner |di-defaultextensions].
```

Comment:
Představ si, že máš nějakou molochovou aplikaci a chceš jí rozdělit do menších nezávislých částí. Pomůžou ti v tom nějak CompilerExtension? To asi dost těžko. Takže bych něco takového vůbec nesliboval.

CompilerExtension neumí nic víc, než upravovat podobu DI kontejneru. Neumí měnit aplikaci, natož ji rozdělovat.